\doxysection{pandas.\+errors.\+Closed\+File\+Error Class Reference}
\hypertarget{classpandas_1_1errors_1_1ClosedFileError}{}\label{classpandas_1_1errors_1_1ClosedFileError}\index{pandas.errors.ClosedFileError@{pandas.errors.ClosedFileError}}


Inheritance diagram for pandas.\+errors.\+Closed\+File\+Error\+:
% FIG 0


Collaboration diagram for pandas.\+errors.\+Closed\+File\+Error\+:
% FIG 1


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Exception is raised when trying to perform an operation on a closed HDFStore file.

Examples
--------
>>> store = pd.HDFStore('my-store', 'a') # doctest: +SKIP
>>> store.close() # doctest: +SKIP
>>> store.keys() # doctest: +SKIP
... # ClosedFileError: my-store file is not open!
\end{DoxyVerb}
 

Definition at line \mbox{\hyperlink{pandas_2errors_2____init_____8py_source_l00674}{674}} of file \mbox{\hyperlink{pandas_2errors_2____init_____8py_source}{\+\_\+\+\_\+init\+\_\+\+\_\+.\+py}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
/home/jam/\+Research/\+IRES-\/2025/dev/src/llm-\/scripts/testing/hypothesis-\/testing/hyp-\/env/lib/python3.\+12/site-\/packages/pandas/errors/\+\_\+\+\_\+init\+\_\+\+\_\+.\+py\end{DoxyCompactItemize}
